protos/IncomingPushMessageSignal.proto

```diff
@@ -114,8 +116,8 @@ message SyncMessage {
   }
 
   message Contacts {
-    optional AttachmentPointer blob       = 1;
-    optional bool              isComplete = 2 [default = false];
+    optional AttachmentPointer blob     = 1;
+    optional bool              complete = 2 [default = false];
```

Comment:
Pretty sure we're not using `complete`/`isComplete` anywhere in the code, so this should be fine.

Reply:
An unsolicited observation:

There are some benefits to changing this file as little as possible. Minimizing drift makes it easier to compare with Signal-Android's which is the de-facto standard, and having a common vocabulary when talking about these inter-client data structures is also nice.

To be fair, iOS has diverged from Android in a couple places too, but generally we make stylistic concessions whenever reasonable to keep the diff small.

Reply:
@michaelkirk All the change in this diff are directly from the java project. :0) 👍

Reply:
well rock on then!

Reply:
Just to confirm, I used `libsignal-service-java`, which I assume is consumed by `Signal-Android`, right?

Reply:
> Just to confirm, I used libsignal-service-java, which I assume is consumed by Signal-Android, right?

yup 👍